app/main/views/g_cloud.py

```diff
@@ -276,6 +276,25 @@ def search_services():
     )
 
 
+@direct_award.route('', methods=['GET'])
+def saved_search_overview():
+    projects = data_api_client.find_direct_award_projects(current_user.id).get('projects', [])
+    open_searches = []
+    closed_searches = []
```

Comment:
'Closed' is going to be confusing for DOS developers, because 'closed' is a status quite similar to our 'locked' status (i.e. it comes before 'awarded').

Can we just call this `active_searches` and `inactive_searches`?